gradoop-common/src/main/java/org/gradoop/common/model/impl/id/GradoopIds.java

```diff
@@ -167,11 +167,14 @@ public boolean contains(GradoopId identifier) {
   /**
    * Checks if the specified ids are contained in the set.
    *
-   * @param ids the ids to look for
+   * @param other the ids to look for
    * @return true, iff all specified ids are contained in the set
    */
-  public boolean containsAll(GradoopIds ids) {
-    for (GradoopId id : ids) {
+  public boolean containsAll(GradoopIds other) {
```

Comment:
We should rename `GradoopIds` to `GradoopIdSet` to make the semantics more clear